CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# Code ownership information.
+# See https://help.github.com/articles/about-code-owners/ for details.
+
+# These owners will be the default owners for everything in the repo. Unless a
+# later match takes precedence, # @trusted-contributors will be requested for
+# review when someone opens a pull request.
+*       @trusted-contributors
+
```

Comment:
Hi, just a formal minor - as the last comment above states "These owners will be the default owners for everything in repo.", maybe we can add here a comment mentioning, that from this point we are listing plugin owners for sake of clarity.
BR, KK

Reply:
Nevermind, I'm late.